CFC_DataCollector/tests/recommenderTests/TestNewPipeline.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import json
+import logging
+import numpy as np
+from get_database import get_db, get_mode_db, get_section_db
+from recommender import pipeline
+import sys
+import os
+
+logging.basicConfig(level=logging.DEBUG)
+
+class TestPipeline(unittest.TestCase):
```

Comment:
This should really be owned by the glue team. We probably want to start with test cases that invoke the individual pipelines end to end first, and let the people writing the modules write test cases for the individual steps in the pipelines....